### DISCUSSION TITLE
Netlify CMS + hugo-PaperMod implement home page settings

### DISCUSSION THREAD

**Discussion by mantans** [adityatelange/hugo-PaperMod#287] [Show and tell]
i have succesfully added Netlify CMS, it works like a charm for posts and about me page. Actually i need to add some specific parts of home page to be edited through the frontend, but i don’t know how to proceed. I have successefully added the Title of website from config.yml but i need to implement the homeInfoParams fields (Title and Content).

Theme config.yml fields i need to implement:

        homeInfoParams:
               Title: My Title Here
               Content: >
                   My Content Here
  
my actually Netlify cms config.yml

       backend:
           name: git-gateway
           branch: master # Branch to update (optional; defaults to master)

       # This line should *not* be indented
       publish_mode: editorial_workflow

      # These lines should *not* be indented
      media_folder: "static/images/uploads" # Media files will be stored in the repo under 
      static/images/uploads
      public_folder: "/images/uploads" # The src attribute for uploaded media will begin with /images/uploads

      collections:
         - name: "posts"
            label: "Posts"
            folder: "content/posts"
            create: true
            slug: "{{year}}-{{month}}-{{day}}-{{slug}}" # Filename template, e.g., YYYY-MM-DD-title.md
            fields:
              - { label: "Title", name: "title", widget: "string" }
              - { label: "Date", name: "date", widget: "date" }
              - { label: "Tags", name: "tags", widget: "list" }
              - { label: "Categories", name: "categories", widget: "string" }
              - { label: "Body", name: "body", widget: "markdown" }
              - { label: "Description", name: "description", widget: "string"}

         - name: "about-me"
            label: "About-me"
            folder: "content/about-me"
            create: true
            slug: "{{year}}-{{month}}-{{day}}-{{slug}}" # Filename template, e.g., YYYY-MM-DD-title.md
            fields:
              - { label: "Title", name: "title", widget: "string" }
              - { label: "Date", name: "date", widget: "date" }
              - { label: "Body", name: "body", widget: "markdown" }

         - name: "settings"
            label: "Settings"
            delete: false # Prevent users from deleting documents in this collection
            editor:
               preview: false
            files:
              - name: 'general'
                 label: 'Site Settings'
                 file: 'config.yml'
                 description: 'General Home Settings'
                 fields:
                   - { label: 'Title', name: 'Title', widget: 'string' }`

**Comment by RoneoOrg** [comment]
You may find interesting bits here: https://stackoverflow.com/questions/48365670/add-to-hugo-site-variables-from-netlify-cms/48371570#48371570

I'm interested in your findings, please report if you get something functionnal!
Cheers
